app/controllers/v0/profile/emails_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module V0
+  module Profile
+    class EmailsController < ApplicationController
+      before_action { authorize :evss, :access? }
+
+      # Fetches the email address, and its effective datetime, for the current user
```

Comment:
All for yard/rdocs but I think in this case swagger covers the response (in this case it's a short response but we have some endpoints that would have an example hundreds of lines long)